app/models/auth_sources/auth_source_ldap.rb

```diff
@@ -75,6 +75,8 @@ def authenticate(login, password)
     logger.debug "Retrieved LDAP Attributes for #{login}: #{attrs}"
 
     attrs
+  rescue Net::LDAP::Error => error
+    raise ::Foreman::LdapException.new(N_("Error connection to %{name} LDAP server with url: %{url} during authentication: %{error}" % {:url => self.host, :name => self.name, :error => error}))
```

Comment:
if ::Foreman::LdapException inherits from wrapped exception, the constructor would accept extra argument - the original exception, `error` in this case

the line would look similar to
`raise ::Foreman::LdapException.new(error, N_("Error while connecting to '%{name}' LDAP server at '%{url}' during authentication" % {:url => self.host, :name => self.name}))`

please note I also update the exception text, it's good to wrap values in some delimiter chars.